### PR TITLE
dev-vcs/repo: add 2.45 (and proxy maintain) 

### PR DESCRIPTION
**Pull request by joecool1029**
Closes: https://bugs.gentoo.org/931692

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-vcs/repo/repo-2.45.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# NB: The ${PV} tracks the *repo launcher version*, not the last signed release
+# of the repo project.  The launcher only gets a new update when changes are
+# made in it.
+
+EAPI="7"
```
Comment (Flowdalic) [inline dev-vcs/repo/repo-2.45.ebuild:8]: Bump to EAPI 8 please

File: dev-vcs/repo/repo-2.45.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+inherit bash-completion-r1 python-r1
+
+DESCRIPTION="Google tool for managing git, particularly multiple repos"
+HOMEPAGE="https://gerrit.googlesource.com/git-repo"
+SRC_URI="https://github.com/GerritCodeReview/git-repo/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE=""
```
Comment (Flowdalic) [inline dev-vcs/repo/repo-2.45.ebuild:21]: Drop empty IUSE

File: dev-vcs/repo/repo-2.45.ebuild (hunk 1 above)
Comment (joecool1029) [reply dev-vcs/repo/repo-2.45.ebuild]: Done, and tested.

**Comment by Flowdalic** [comment]
Thanks, bonus points if you squash the latest commit into the first one.

**Pull request closed without merging**